### ISSUE TITLE
Testing dataset for training

### ISSUE THREAD

**Issue by chaohuang** [lyq133/LIC-HPCM#8]
Hi Yuqi,

I notice that one of the input options for `train.py` is `--test_dataset`. As I understand it, the actual testing data should not be used for training. For example, the Kodak dataset should not be used during training.

So IMO, the input option `--test_dataset` for `train.py` should actual be validation dataset, and I wonder what data you used as `--test_dataset` to generate the results in your paper.

Thanks.

**Comment by lyq133** [comment]
Hi! You are absolutely correct. In the context of train.py, the --test_dataset argument functions as a validation set.

I used a subset of the DIV2K validation set for training progress. Since overfitting is typically not a significant issue in learned image compression, the specific choice of validation dataset is generally not critical. You can also feel free to choose any dataset you prefer for validation.

Thanks for pointing this out!
